File: clients/dotnet/enums/NexusTypeId.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// 
    /// </summary>
    public enum NexusTypeId
    {
        /// <summary>
        /// 
        /// </summary>
        None,

        /// <summary>
        /// 
        /// </summary>
        SalesOrSellersUseTax,

        /// <summary>
        /// 
        /// </summary>
        SalesTax,

        /// <summary>
        /// 
        /// </summary>
        SSTVolunteer,

        /// <summary>
        /// 
        /// </summary>
        SSTNonVolunteer,


    }
}

File: clients/php/enums/NexusTypeId.php
<?php
/*
 * AvaTax Enum Class
 *
 * (c) 2004-2016 Avalara, Inc.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
namespace Avalara.AvaTax;

/**
 * @author Ted Spence <ted.spence@avalara.com>
 * @author Bob Maidens <bob.maidens@avalara.com>
 */class NexusTypeId extends AvaTaxEnum 
{

    const None = "None";
    const SalesOrSellersUseTax = "SalesOrSellersUseTax";
    const SalesTax = "SalesTax";
    const SSTVolunteer = "SSTVolunteer";
    const SSTNonVolunteer = "SSTNonVolunteer";
}
